contracts/Strategy.sol

```diff
@@ -80,6 +80,15 @@ contract Strategy is BaseStrategy {
         targetRatioMultiplier = _targetRatioMultiplier;
     }
 
+    function migrateSusdVault(IVault _sUSDVault) external onlyGovernance {
+        uint256 _balanceToWithdraw = balanceOfSusdInVault();
+        withdrawFromSUSDVault(_balanceToWithdraw);
```

Comment:
why not `susdVault.withdraw()` directly?

Reply:
i wanted to use same function that in the rest of the code. it is not gas optimised but this function will be rarely used. what do you think?

Reply:
I would use `susdVault.withdraw()` since we are withdrawing all no matter what.

Reply:
amen